docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -95,42 +96,106 @@ \section{Team Member Roles}
+
+    \subsection*{Proof of Concept Demonstration}
+    There might be many moving parts to our idea, but once the basis for them is set up, a lot of them will be managed a lot more smoothly moving forward. In order for us to feel as though our project is moving forward at an acceptable rate, there are a few basic milestones we would like to reach:
+
+    \begin{itemize}
+        \item Have a simple data storage, even if it's as basic as reading and writing from a test file for now
+        \item A workflow that can extract basic data from a user's syllabus
+        \item Preliminary model that accepts syllabus and schedule data
+        \item Template pipeline with steps and framework chosen to handle the model training
+        \item Host a local-based web app that can handle some basic communication between the model and the database 
+        \item Minimal user interface designed simply for functionality
+    \end{itemize}
+
+    Setting these up will be the bulk of the learning curve for technologies that aren't our main focus. Setting up a form of simple storage database, website, and data extraction workflow, are all processes that don't need to be altered too much while working with the data. Getting all these done and out of the way at this point will let us be able to focus on the lengthy process that will be fine-tuning the model and training pipeline to try to get the best results from our data.
+
+    \subsection*{Will testing be difficult?}
+    Testing the model's performance will not be difficult as some of us have worked on AI projects that perform cross-validation (K-fold) tests using \textit{scikit-learn} library. For those members that are not familiar with cross-validation testing or \textit{scikit-learn} library, online documentations and tutorials can be found easily for example, accessing \textit{scikit-learn} website from \href{https://scikit-learn.org/stable/index.html}{here}, reading cross-validation documentation from \href{https://scikit-learn.org/stable/modules/cross_validation.html}{here}.
+    Testing web apps will not be difficult as we have worked with \textit{Python} projects that are using \textit{Streamlit/Flask} as the web framework and \textit{PyTest} as the testing framework from previous co-op work terms and have learned and used those frameworks from projects of previous courses like \textit{SFWRENG 3X03}. As a result, we will be able to perform unit tests and integration tests for the web apps. 
+    \subsection*{Is a required library difficult to install?}
+    No, the required library is not difficult to install, using tools like pip can easily install the library required very easily by running a command line.
```

Comment:
use \texttt{pip} instead?

Reply:
Yeah, but using \textit{} to be consistent

Reply:
next time could you please click on the line that contains the part that needs to be changed instead of using multi-lines selection? Thanks!